Assignment1/test/test_email_verifier.py

```diff
@@ -57,3 +57,8 @@ def test_empty_string(em):
 
 def test_non_string(em):
     assert not em.is_valid_email(42)
+
+
+def test_accepts_only_3_character_tlds(em):
+    assert em.is_valid_email("jacob@jacobmason.valid.net")
```

Comment:
Needs to test against 3 or fewer character tlds. Fails on my local test starting with 2 character tlds.